roofit/roofitcore/src/RooAbsMinimizerFcn.cxx

```diff
@@ -0,0 +1,554 @@
+/*****************************************************************************
+ * Project: RooFit                                                           *
+ * Package: RooFitCore                                                       *
+ * @(#)root/roofitcore:$Id$
+ * Authors:                                                                  *
+ *   AL, Alfio Lazzaro,   INFN Milan,        alfio.lazzaro@mi.infn.it        *
+ *   PB, Patrick Bos, Netherlands eScience Center, p.bos@esciencecenter.nl   *
+ *                                                                           *
+ *                                                                           *
+ * Redistribution and use in source and binary forms,                        *
+ * with or without modification, are permitted according to the terms        *
+ * listed in LICENSE (http://roofit.sourceforge.net/license.txt)             *
+ *****************************************************************************/
+
+//////////////////////////////////////////////////////////////////////////////
+//
+// RooAbsMinimizerFcn is an interface class to the ROOT::Math function
+// for minimization. It contains only the "logistics" of synchronizing
+// between Minuit and RooFit. Its subclasses implement actual interfacing
+// to Minuit by subclassing IMultiGenFunction or IMultiGradFunction.
+//
+
+#include "RooAbsMinimizerFcn.h"
+
+#include "RooAbsArg.h"
+#include "RooAbsPdf.h"
+#include "RooArgSet.h"
+#include "RooRealVar.h"
+#include "RooAbsRealLValue.h"
+#include "RooMsgService.h"
+#include "RooMinimizer.h"
+#include "RooNaNPacker.h"
+
```

Comment:
```suggestion
#include "TClass.h"
```

Because of the `arg->IsA()->InheritsFrom()` in the same file below. Unfortunately ROOT doesn't build on my system before this change.

Reply:
Weird how those includes work differently on different systems sometimes... My IDE (CLion on macOS) even immediately notes that the include is unused. Perhaps something to do with platform-based compile flags then? Anyway, fixed!